pandas/io/excel/_base.py

```diff
@@ -101,10 +102,15 @@
     If io is not a buffer or path, this must be set to identify io.
     Supported engines: "xlrd", "openpyxl", "odf", "pyxlsb", default "xlrd".
     Engine compatibility :
-    - "xlrd" supports most old/new Excel file formats.
```

Comment:
Can you add a blank line above this one? (then the sphinx rendering will actually work)